tests/test_store.py

```diff
@@ -432,27 +466,50 @@ def test_getDatasetManifest(self, synapse_store, downloadFile):
             assert manifest_data == "syn51204513"
 
     @pytest.mark.parametrize(
-        "existing_manifest_df",
+        "existing_manifest_df,fill_in_return_value",
         [
-            pd.DataFrame(),
-            pd.DataFrame(
-                {
-                    "Filename": ["existing_mock_file_path"],
-                    "entityId": ["existing_mock_entity_id"],
-                }
+            (
+                pd.DataFrame(),
+                [
+                    {
+                        "Filename": ["mock_file_path"],
+                        "entityId": ["mock_entity_id"],
+                    },
+                    {
+                        "Filename": ["mock_file_path"],
+                        "entityId": ["mock_entity_id"],
+                    },
+                ],
+            ),
+            (
+                pd.DataFrame(
+                    {
+                        "Filename": ["existing_mock_file_path"],
+                        "entityId": ["existing_mock_entity_id"],
+                    }
+                ),
+                [
+                    {
+                        "Filename": ["existing_mock_file_path", "mock_file_path"],
+                        "entityId": ["existing_mock_entity_id", "mock_entity_id"],
+                    },
+                    {
+                        "Filename": ["mock_file_path"],
+                        "entityId": ["mock_entity_id"],
+                    },
+                ],
```

Comment:
Also, later in the test, you have: 
```

            if not existing_manifest_df.empty:
                expected_df = pd.DataFrame(
                    {
                        "Filename": ["existing_mock_file_path", "mock_file_path"],
                        "entityId": ["existing_mock_entity_id", "mock_entity_id"],
                    }
                )
            else:
                expected_df = pd.DataFrame(
                    {"Filename": ["mock_file_path"], "entityId": ["mock_entity_id"]}
                )
```
I think the "expected_df" can be in @pytest.mark.parametrize. So you should just need to do: 
`assert_frame_equal(new_manifest, expected_df)` without doing if else